assets/js/list-comments.js

```diff
@@ -0,0 +1,170 @@
+            prevText: '<i class="material-icons">chevron_left</i>',
+            nextText: '<i class="material-icons">chevron_right</i>',
+            useAnchors: false,
+            displayedPages: 2,
+            onPageClick: paginateComments
+          });
+        } else {
+          $('#product-comments-list-pagination').hide();
+        }
+      } else {
+        commentsList.html('');
+        emptyProductComment.show();
+        commentsList.append(emptyProductComment);
+      }
+    });
+  }
+
+  function populateComments(comments) {
+    commentsList.html('');
+    comments.forEach(addComment);
+  }
+
+  function addComment(comment) {
+    var commentTemplate = commentPrototype;
+    var customerName = comment.customer_name;
+    if (!customerName) {
+      customerName = comment.firstname+' '+comment.lastname;
+    }
+    commentTemplate = commentTemplate.replace(/@COMMENT_ID@/, comment.id_product_comment);
+    commentTemplate = commentTemplate.replace(/@PRODUCT_ID@/, comment.id_product);
+    commentTemplate = commentTemplate.replace(/@CUSTOMER_NAME@/, customerName);
+    commentTemplate = commentTemplate.replace(/@COMMENT_DATE@/, comment.date_add);
+    commentTemplate = commentTemplate.replace(/@COMMENT_TITLE@/, comment.title);
+    commentTemplate = commentTemplate.replace(/@COMMENT_COMMENT@/, comment.content);
+    commentTemplate = commentTemplate.replace(/@COMMENT_USEFUL_ADVICES@/, comment.usefulness);
+    commentTemplate = commentTemplate.replace(/@COMMENT_NOT_USEFUL_ADVICES@/, (comment.total_usefulness - comment.usefulness));
+    commentTemplate = commentTemplate.replace(/@COMMENT_TOTAL_ADVICES@/, comment.total_usefulness);
+
+    const $comment = $(commentTemplate);
+    $('.grade-stars', $comment).rating({
+      grade: comment.grade
+    });
+    $('.useful-review', $comment).click(function() {
+      updateCommentUsefulness($comment, comment.id_product_comment, 1);
+    });
+    $('.not-useful-review', $comment).click(function() {
+      updateCommentUsefulness($comment, comment.id_product_comment, 0);
+    });
+    $('.report-abuse', $comment).click(function() {
+      confirmCommentAbuse(comment.id_product_comment);
+    });
+
+    commentsList.append($comment);
+  }
+
+  function updateCommentUsefulness($comment, commentId, usefulness) {
+    $.post(updateCommentUsefulnessUrl, {id_product_comment: commentId, usefulness: usefulness}, function(jsonResponse){
+      var jsonData = false;
+      try {
+        jsonData = JSON.parse(jsonResponse);
+      } catch (e) {
+      }
+      if (jsonData) {
+        if (jsonData.success) {
+          $('.useful-review-value', $comment).html(jsonData.usefulness);
+          $('.not-useful-review-value', $comment).html(jsonData.total_usefulness - jsonData.usefulness);
+        } else {
+          showUpdatePostCommentErrorModal(jsonData.error);
+        }
+      } else {
+        showUpdatePostCommentErrorModal('Sorry, your review appreciation could not be sent.');
```

Comment:
Is it a wording?

Reply:
Yes it is! You're right I should inject a translated message in the template.